sdk/java-sdk/src/main/resources/reference.conf

```
@@ -16,13 +16,21 @@ kalix {
   }
 
   discovery {
-    // The path to a file on the classpath that contains a compiled descriptor that contains source info.
-    // This descriptor should have been compiled by protobuf using the --descriptor_set_out and
-    // --include_source_info flags. If this file is present, the Java SDK will use this to fill in the source
-    // info for descriptors it sends to the proxy during discovery, since the descriptors compiled into the
-    // generated Java source do not include source info. To disable use of such a descriptor for locating
-    // source info, set this to "disabled".
+    # The path to a file on the classpath that contains a compiled descriptor that contains source info.
+    # This descriptor should have been compiled by protobuf using the --descriptor_set_out and
+    # --include_source_info flags. If this file is present, the Java SDK will use this to fill in the source
+    # info for descriptors it sends to the proxy during discovery, since the descriptors compiled into the
+    # generated Java source do not include source info. To disable use of such a descriptor for locating
+    # source info, set this to "disabled".
     protobuf-descriptor-with-source-info-path = "/protobuf/descriptor-sets/user-function.desc"
+
+    # By default all environment variables of the process is passed along to the proxy, they are used only for
+    # substituion usage in the descriptor options such as topic names. By default all environment is passed along,
```

```suggestion
    # substitution usage in the descriptor options such as topic names.
```

Was "By default all environment is passed along," a leftover maybe?